### PR TITLE
feat: add 2captcha support

### PR DESCRIPTION
**Pull request by MrlolDev**
Add support for 2captcha

### CODE REVIEW

**Comment by transitive-bullshit** [comment]
Thanks @MrlolDev 🙏 It's great recognizing people from discord on here 😄 

I'm traveling right now, so I won't have a chance to take a proper look at this til later tomorrow, but just wanted to let you know that I really appreciate the PR.

**Comment by marcorizza** [comment]
solveRecaptchas method does not wait until it has finished solving and goes ahead in the process by clicking the submit button

File: src/openai-auth-2captcha.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+}: {
+  email?: string
+  password?: string
+  timeoutMs?: number
+  browser?: Browser
+  captchaToken?: string
+}): Promise<OpenAIAuth> {
+  let page: Page
+  let origBrowser = browser
+  puppeteer.use(
```
Comment (transitive-bullshit) [inline src/openai-auth-2captcha.ts:46]: will this add the plugin again every time `getOpenAIAuth2Captcha` is called?

**Comment by transitive-bullshit** [comment]
@MrlolDev @marcorizza I'm back from traveling now, so more time to experiment. I'm going to merge this and then experiment with things quite a bit before pushing out any sort of release.

Super appreciated! 🙏

**Pull request merged**